4-projects/business_unit_1/development/README.md

```diff
@@ -18,12 +18,14 @@
 | base\_subnets\_self\_links | The self-links of subnets from base environment. |
 | bucket | The created storage bucket |
 | env\_secrets\_project | Project sample peering project id. |
+| firewall\_tags | The security tags created for firewall. |
```

Comment:
I think the description needs to provide mor information regarding which type of tags these are and the context they should be used. They look like generic tags but they can only be used for the peering network
the information that they are for the peering network must be in the name and in the description